src/test/java/lingogo/logic/commands/ExportCommandTest.java

```diff
@@ -56,40 +59,63 @@ public void execute_slideshowActive_throwsCommandException() {
 
     @Test
     public void execute_exportTypicalFlashcards_successfulExport() {
+        createDataFolder();
         String fileName = "exportTest.csv";
         String expectedMessage = String.format(ExportCommand.MESSAGE_SUCCESS, fileName);
         ExportCommand command = new ExportCommand(fileName);
         try {
-            expectedModel.exportFlashCards(fileName);
+            command.exportHelper(expectedModel);
         } catch (Exception e) {
             fail("Exception not expected");
         }
         assertCommandSuccess(command, model, expectedMessage, expectedModel);
```

Comment:
I think can just use `model` as the `expectedModel`, since exporting shouldn't change the `model` at all right?